.travis.yml

```diff
@@ -192,6 +192,13 @@ before_install:
               tfold ext.mongodb tpecl mongodb-1.6.0 mongodb.so $INI
           elif [[ $PHP = nightly ]]; then
               tfold ext.memcached tpecl memcached-3.1.5 memcached.so $INI
+              tfold ext.apcu wget https://github.com/krakjoe/apcu/archive/9c36db45100d4d27ec33072f4be90f1f5a0108b7.zip \
```

Comment:
tfold does work this way, see output in travis
we might need a bash function I thunk

Reply:
done